3-platform/warehouse/spec/src/main/java/com/_6core/platform/warehouse/rest/v1/ProductModificationSpecification.java

```diff
@@ -0,0 +1,84 @@
+package com._6core.platform.warehouse.rest.v1;
+
+import com._6core.platform.warehouse.rest.v1.dto.ProductResponseDto;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.parameters.RequestBody;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import reactor.core.publisher.Mono;
+
+import java.math.BigDecimal;
+
+/**
+ * Interface for modifying product information in the Warehouse Service.
+ */
+
+@Tag(name = "Product Modification", description = "REST specification for modifying product information")
+public interface ProductModificationSpecification {
```

Comment:
Discuss with team members about how you name all `Specification` classes.

As I see, Kateryna is called this like `Api`, you call it `Specification`. 
It violates Clean Code principles (choose one name for similar operations). 
```suggestion
public interface ProductModificationSpecification {
```